packages/wallet/src/services/DRepRegistrationTracker.ts

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable sonarjs/cognitive-complexity */
+import * as Crypto from '@cardano-sdk/crypto';
+import { Cardano } from '@cardano-sdk/core';
+import { Observable, map, mergeMap } from 'rxjs';
+import { TrackerSubject } from '@cardano-sdk/util-rxjs';
+
+export interface CreateDRepRegistrationTrackerProps {
```

Comment:
I would suggest not exporting it unless necessary.

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1192/commits/f905e90f5455c5f9df42a4314438796acf479da0#diff-3daae4f884d09049873c2f3c2a5567a39ae0a43af64398a6473983c82c5e0745R7